dplace_app/static/partials/navbar.html

```diff
@@ -34,20 +34,20 @@
 		<div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
 			<ul class="nav navbar-nav navbar-right">
 				<li ng-class="searchActive" class="searchActive">
-					<a ng-href href="/search">
+					<a ng-href="/search">
```

Comment:
@Bibiko Also, I don't know why, but using "ng href href" isn't working on my machine anymore. Maybe it's just me, in which case feel free to ignore this, but why can't we just use "ng-href ="/URL"?